src/version.ts

```diff
@@ -1,4 +1,4 @@
 // Project versions
 // Needs to be updated manually since in client-side, we don't have access to the package.json file.
-export const PLUGGY_JS_VERSION = '0.9.0'
+export const PLUGGY_JS_VERSION = '0.9.1'
```

Comment:
this should be **at least** a minor release, rather than patch. I'm also considering a major release would be reasonable, but maybe we can leave that for when we remove the no longer needed methods we've discussed already, and apply the changes to make this only work with a provided connectToken